scenario/cold_migrate_autorollback.yaml

```diff
@@ -0,0 +1,116 @@
+# This scenario is the same as cold_migrate.yaml plus
+# includes autorollback functionality at the end.
+# This scenario can be used for testing purposes and will leave
+# destination cloud at the same stage as it was before migration.
+
+namespace:
+  info_result:
+      instances: {}
+
+preparation:
+  - pre_migration_test:
+      - act_get_filter: True
+      - act_check_filter: True
+      - check_config_quota_neutron: True
+      - src_test:
+          - act_check_ident_api_src: True
+          - act_check_image_api_src: True
+          - act_check_compute_api_src: True
+          - act_check_storage_api_src: True
+          - act_check_neutron_api_src: True
+          - check_src_ssh_access: True
+          - act_check_bandwidth_src: True
+          - check_src_sql: True
+          - check_src_rabbit: True
+          - act_get_info_objects_src: False
+      - dst_test:
+          - act_check_ident_api_dst: True
+          - act_check_image_api_dst: True
+          - act_check_compute_api_dst: True
+          - act_check_storage_api_dst: True
+          - act_check_neutron_api_dst: True
+          - check_dst_ssh_access: True
+          - act_check_bandwidth_dst: True
+          - check_dst_sql: True
+          - check_dst_rabbit: True
+          - act_get_info_objects_dst: False
+      - check_networks: True
+  - check_users_availability: False
```

Comment:
Do we need this task here? If yes, let us set it in `True`, if no, let us remove it from here.